extras/uiuc-2016-04/README.md

```diff
@@ -0,0 +1,5 @@
+# Sample Graders #
+
+Brennan Saeta <@saeta> authored these graders as part of a grader-authoring workshop conducted at
+UIUC in April 2016. See [the slides](https://docs.google.com/presentation/d/1qRU7HM5JBFH3wtTW01GRZrs5X-EtL_HRjY-aDtqqiSc/present)
+for further background.
```

Comment:
Good comment. I've added a bit more in this README to fill in a bit more what's going on.